### ISSUE TITLE
Pure gauge and glueball spectrum

### ISSUE THREAD

**Issue by mfrasca60** [milc-qcd/milc_qcd#59]
Hi there,

I have compiled the MILC code on my machine to work with pure gauge and gluon propagator. I have generated the input files for gluon_prop by su3_hmc and then I tried to plot what I get from the output file of the former (the data appear to be marked by the labels START_GLUON_PROP and END_GLUON_PROP). What I get is something that does not agree with known results as one can see from e.g. https://arxiv.org/pdf/1611.03642.pdf (see fig.2) and my enclosed figure. It is also difficult for me to see how could a glueball spectrum be obtained similarly to what is seen in e.g. https://inspirehep.net/files/377a31b7e706b0700453ec49b05865a2. Do some set of utilities exist to accomplish such tasks? Am I missing something about gauge fixing?
![prop1](https://github.com/milc-qcd/milc_qcd/assets/130444634/4538f010-bff7-4756-bfa8-abb435a33e02)

**Comment by daschaich** [comment]
I've never touched the `gluon_prop` application, but I see it has a `test` subdirectory.  Are you able to reproduce the sample output in `gluon_prop/test`?  If not, then that's something that others could try to reproduce and diagnose.  If so, then that suggests the problem may be with the input configurations or the output post-processing.

**Comment by mfrasca60** [comment]
Yes, the tests worked fine. I have also plotted the propagator data in the output test files. The possibility, in agreement with what you say, is that the pure_gauge did not give proper data to work with gluon_prop. Please, note that also the tests in pure_gauge worked fine (I have obtained agreement to the decimals). I have QUDA running to perform all these computations.
